readthedocs/projects/migrations/0136_addons_customscript_notnull.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 4.2.16 on 2024-11-13 13:36
+
+from django.db import migrations, models
+from django_safemigrate import Safe
+
+
+class Migration(migrations.Migration):
+    safe = Safe.after_deploy
```

Comment:
Nice making them nullable 💯

Reply:
customscript_enabled is not nullable, so this will still break when creating a new addons object.

Reply:
The previous migration makes it nullable during deploy, and then marks it not nullable after, which I thought would work?

Reply:
ah, I didn't see there were two migrations. I guess Django will ask for a default value when running this migration (for the objects that were created while the field was null).